### ISSUE TITLE
Speed bonus is not implemented

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#860]
According to [§36](http://www.zerocracy.com/policy.html#36) a user should get 5 extra minutes if he completed a job less than 48 hours.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.zerocracy.pm.Claims[107] java.lang.IllegalStateException: Duplicate claims are not allowed, can't add this: <?xml version="1.0" encoding="UTF-8"?>
<claim id="821249003">
   <created>2018-03-23T12:49:27.08Z</created>
   <type>Add job to WBS</type>
   <token>github;zerocracy/farm;860</token>
   <params>
      <param name="job">gh:zerocracy/farm#860</param>
      <param name="reason">GitHub label was attached</param>
      <param name="quiet">true</param>
   </params>
</claim>

```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
Job [#860](https://github.com/zerocracy/farm/issues/860) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by proshin-roman** [comment]
@0crat wait until pull request #1136 is merged

**Comment by 0crat** [comment]
> @0crat wait until pull request #1136 is merged ([here](https://github.com/zerocracy/farm/issues/860#issuecomment-396589573))

@proshin-roman The impediment for [#860](https://github.com/zerocracy/farm/issues/860) was registered successfully by @proshin-roman[/z](https://www.0crat.com/u/proshin-roman)

**Comment by proshin-roman** [comment]
@0crat refuse

**Comment by proshin-roman** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/860#issuecomment-402048312))

@proshin-roman The job [#860](https://github.com/zerocracy/farm/issues/860) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), you cannot resign, since you are not a `PO` or `ARC`

**Comment by amihaiemil** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/860#issuecomment-402052059))

@amihaiemil This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#860](https://github.com/zerocracy/farm/issues/860) is [in scope](http://www.zerocracy.com/policy.html#14) for 3mon
  * The role is `DEV`
  * The job is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) for [14hr](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) will get 50% of their hourly rate for this job
  * Some equity will be [vested](http://www.zerocracy.com/policy.html#37) on completion
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @g4s8[/z](https://www.0crat.com/u/g4s8): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23860%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by proshin-roman** [comment]
@g4s8 I still see this ticket in my agenda (on hold status). Don't think I have to submit a new bug as there was a known problem some time ago with the bot. Could you fix it, please? I wanted to refuse this ticket.

**Comment by g4s8** [comment]
@proshin-roman this job is not assigned to you, your agenda will be updated soon automatically

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by amihaiemil** [comment]
@0crat wait for review of PR #1379

**Comment by 0crat** [comment]
> @0crat wait for review of PR #1379  ([here](https://github.com/zerocracy/farm/issues/860#issuecomment-403289770))

@amihaiemil The impediment for [#860](https://github.com/zerocracy/farm/issues/860) was registered successfully by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
> @0crat wait for review of PR #1379  ([here](https://github.com/zerocracy/farm/issues/860#issuecomment-403289770))

@amihaiemil Job [#860](https://github.com/zerocracy/farm/issues/860) is not in the agenda of @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), can't set impediment

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1381](https://github.com/zerocracy/farm/issues/1381) is still not solved.

**Comment by amihaiemil** [comment]
@g4s8 PR #1379 was merged, can we cloae?

**Comment by g4s8** [comment]
@amihaiemil sure, thanks

**Comment by 0crat** [comment]
Job [#860](https://github.com/zerocracy/farm/issues/860) is not in the agenda of @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), can't inspect

**Comment by 0crat** [comment]
The job [#860](https://github.com/zerocracy/farm/issues/860) is now out of scope

**Comment by amihaiemil** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/860#issuecomment-403413930))

@amihaiemil This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#860](https://github.com/zerocracy/farm/issues/860) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 16min
  * @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) will receive 30 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23860%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ypshenychka** [comment]
@amihaiemil Thanks! Looking into it.

**Comment by ypshenychka** [comment]
@amihaiemil According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>All commit messages have links to issues

[One](https://github.com/zerocracy/farm/pull/1379/commits/a1352959c67217e44f9ff438757c212285782818) doesn't contain link to issue. Please confirm you'll try to follow this rule next time.

**Comment by amihaiemil** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@amihaiemil Thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +30 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#1483](https://github.com/zerocracy/farm/issues/1483), [#1484](https://github.com/zerocracy/farm/issues/1484) are still not solved; solved: [#1381](https://github.com/zerocracy/farm/issues/1381).
